syncer/atxsync/atxsync.go

```diff
@@ -50,12 +51,16 @@ func Download(
 		logger.Info("downloaded atxs",
 			zap.Int("total", total),
 			zap.Int("downloaded", downloaded),
-			zap.Array("missing", zapcore.ArrayMarshalerFunc(func(enc zapcore.ArrayEncoder) error {
-				for _, atx := range missing {
-					enc.AppendString(atx.ShortString())
-				}
-				return nil
-			})))
+			zap.Float64("progress [%]", float64(downloaded)/float64(total)*100),
```

Comment:
Has the advantage of being able to do it with integers than floats and automatically rounds it to full %

```suggestion
			zap.Float64("progress [%]", downloaded*100/total),
```